docs/changes/coordinates/11943.bugfix.rst

```diff
@@ -0,0 +1 @@
+NaN values are now printed for Angle, Latitude and Longitude in degrees.
```

Comment:
Perhaps,

"Fixed bug where Angle, Latitude and Longitude with NaN values could not be printed"

Reply:
✅